build.gradle.kts

```diff
@@ -14,10 +14,6 @@ plugins {
     id("toolkit-jacoco-report")
 }
 
-group = "software.aws.toolkits"
```

Comment:
group?

Reply:
We never used it it seems

I can add it for the sake of it though?

Reply:
To be clear, we didnt use it because we dont publish anything